Documentation/GeneralConventions/GuidesXml.rst

```diff
@@ -0,0 +1,197 @@
+..  _guides-xml:
+
+================================
+:file:`Documentation/guides.xml`
+================================
+
+This XML file contains meta information and configuration used during rendering
+of a manual.
+
+..  hint::
+    If you are migrating from the legacy Sphinx based rendering and still have
+    a :file:`Documentation/settings.cfg` you can use an automatic migration
+    tool to :ref:`migrate the settings.cfg into a guides.xml <migrate_guides_xml>
+
+.. _settings-guides:
+
+The `<guides>` tag
+==================
+
+Is used for configuration during parsing and rendering. Most available settings
+are predefined in the rendering chain supplied by the
+:ref:`TYPO3 Rendering Container <rendering-docs>`.
```

Comment:
"TYPO3 Rendering Container" -> "Documentation Rendering Container"

TYPO3 is not rendered.